src/main/java/com/joelj/jenkins/eztemplates/TemplateImplementationProperty.java

```diff
@@ -1,26 +1,38 @@
 package com.joelj.jenkins.eztemplates;
 
+import java.util.Collection;
+import java.util.List;
+import java.util.logging.Logger;
+
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.export.Exported;
+
 import com.google.common.collect.ImmutableList;
-import com.joelj.jenkins.eztemplates.exclusion.*;
+import com.joelj.jenkins.eztemplates.exclusion.AssignedLabelExclusion;
+import com.joelj.jenkins.eztemplates.exclusion.DescriptionExclusion;
+import com.joelj.jenkins.eztemplates.exclusion.DisabledExclusion;
+import com.joelj.jenkins.eztemplates.exclusion.Exclusion;
+import com.joelj.jenkins.eztemplates.exclusion.Exclusions;
+import com.joelj.jenkins.eztemplates.exclusion.EzTemplatesExclusion;
+import com.joelj.jenkins.eztemplates.exclusion.JobParametersExclusion;
+import com.joelj.jenkins.eztemplates.exclusion.MatrixAxisExclusion;
+import com.joelj.jenkins.eztemplates.exclusion.ScmExclusion;
+import com.joelj.jenkins.eztemplates.exclusion.TriggersExclusion;
+import com.joelj.jenkins.eztemplates.jobtypes.JobsFacade;
```

Comment:
It's better your way but we'll do it separately. I have a load of housekeeping changes I'll do when I'm certain there are no PRs out there.